[2023-10-04T07:42:37.077+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-10-03T00:00:00+00:00 [queued]>
[2023-10-04T07:42:37.123+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-10-03T00:00:00+00:00 [queued]>
[2023-10-04T07:42:37.132+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-10-04T07:42:37.137+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-10-04T07:42:37.140+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-10-04T07:42:37.207+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-10-03 00:00:00+00:00
[2023-10-04T07:42:37.220+0000] {standard_task_runner.py:55} INFO - Started process 76 to run task
[2023-10-04T07:42:37.231+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-10-03T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmphjgoudq_']
[2023-10-04T07:42:37.238+0000] {standard_task_runner.py:83} INFO - Job 32: Subtask create_table
[2023-10-04T07:42:37.457+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-10-03T00:00:00+00:00 [running]> on host f637e1d287c6
[2023-10-04T07:42:37.658+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-10-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-03T00:00:00+00:00
[2023-10-04T07:42:37.697+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-10-04T07:42:37.722+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-10-04T07:42:37.733+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-10-04T07:42:37.764+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20231003T000000, start_date=20231004T074237, end_date=20231004T074237
[2023-10-04T07:42:37.843+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-10-04T07:42:37.935+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
